javascript/powercommands/command_runner.js

```diff
@@ -0,0 +1,64 @@
+const commands = {
```

Comment:
Code structure: consider making each command a class that knows how to parse/execute itself.